source/instructors/setupcourses/computedvm.rst

```diff
@@ -0,0 +1,82 @@
+.. meta::
+   :description: Instructions for using Computed VMs.
+
+
+.. _computedvm:
+
+Computed VMs
+============
+
+Codio supports Windows OS in **Computed VM** boxes . To enable this for your organisation, send an email to help@codio.com with approximate details of the number of students who will require access to computed VMs.
+
+.. Note:: Codio are not responsible for licensing issues in regard to any software you install to use.
```

Comment:
Line 12

Codio is not responsible for any licensing issues related to the software you install.